inventory/byo/hosts.example

```diff
@@ -24,7 +24,7 @@ deployment_type=atomic-enterprise
 #use_cluster_metrics=true
 
 # Pre-release registry URL
-#oreg_url=rcm-img-docker01.build.eng.bos.redhat.com:5001/openshift3/ose-${component}:${version}
+#oreg_url=registry.access.redhat.com:5001/openshift3/ose-${component}:${version}
```

Comment:
I'd leave it as is as it's technically correct however that registry isn't accessible externally. Perhaps we should change this to be
```
# Alternate image format string, useful for internal mirrors
#oreg_url=example.com/openshift3/ose-${component}:${version}
```